tasks/check-usb-devices.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: List USB hardware
+  command: lsusb  -d {{ usb_device.vendor }}:{{ usb_device.product }}
+  register: host_attached_usb_device
+  become: true
+  changed_when: false 
+  ignore_errors: true
```

Comment:
nit: I prefer `failed_when: false` since it doesn't show as failed

Reply:
Cool I will update that later today - Agreed about not having it failed twice on this section and it will be cleaner.

Reply:
Changed to failed_when in commit https://github.com/stackhpc/ansible-role-libvirt-vm/pull/95/commits/20998ddf4617303b1d8a007da68e851bb06ebfb6

This is a lot cleaner now

```
TASK [ansible-role-libvirt-vm : List USB hardware] **********************************************************************************************************************************************************
task path: /Users/gavin/Documents/GIT/HomeLab/ansible/roles/ansible-role-libvirt-vm/tasks/check-usb-devices.yml:2
ok: [kvm.gavinwill.me.uk] => {"changed": false, "cmd": ["lsusb", "-d", "0x0781:0x5566"], "delta": "0:00:00.010150", "end": "2023-03-13 11:42:19.654463", "failed_when_result": false, "msg": "non-zero return code", "rc": 1, "start": "2023-03-13 11:42:19.644313", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}

TASK [ansible-role-libvirt-vm : Check USB device is present on Host system] *********************************************************************************************************************************
task path: /Users/gavin/Documents/GIT/HomeLab/ansible/roles/ansible-role-libvirt-vm/tasks/check-usb-devices.yml:9
fatal: [kvm.gavinwill.me.uk]: FAILED! => {"changed": false, "msg": "The USB Device with Vendor ID:0x0781 and Product ID:0x5566 is not seen on host system Is the USB device plugged in correctly ?\n"}

PLAY RECAP **************************************************************************************************************************************************************************************************
kvm.gavinwill.me.uk        : ok=21   changed=1    unreachable=0    failed=1    skipped=15   rescued=0    ignored=0   
```